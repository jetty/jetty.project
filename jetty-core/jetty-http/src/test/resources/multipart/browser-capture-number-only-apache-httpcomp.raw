--xE8WoYDcbqAfj08bxPk669iK22hMMlZL
Content-Disposition: form-data; name="pi"

3.14159265358979323846264338327950288419716939937510
--xE8WoYDcbqAfj08bxPk669iK22hMMlZL--
