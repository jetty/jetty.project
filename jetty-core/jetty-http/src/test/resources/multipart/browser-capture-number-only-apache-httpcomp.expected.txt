Request-Header|Accept-Encoding|gzip,deflate
Request-Header|Connection|keep-alive
Request-Header|Content-Length|173
Request-Header|Content-Type|multipart/form-data; boundary=xE8WoYDcbqAfj08bxPk669iK22hMMlZL
Request-Header|Host|localhost:9090
Request-Header|User-Agent|Apache-HttpClient/4.5.5 (Java/1.8.0_162)
Request-Header|X-BrowserId|apache-httpcomp
Parts-Count|1
Part-ContainsContents|pi|3.14159265358979323846264338327950288419716939937510
